packages/batch-submitter/src/batch-submitter/state-batch-submitter.ts

```diff
@@ -122,7 +151,11 @@ export class StateBatchSubmitter extends BatchSubmitter {
       const block = (await this.l2Provider.getBlockWithTransactions(
         i
       )) as L2Block
-      batch.push(block.stateRoot)
+      if (block.transactions[0].from === this.fraudSubmissionAddress) {
+        batch.push(ethers.utils.keccak256(Date.now().toString(16)))
```

Comment:
Could we push a specific root instead of a hash? Would be nice to be able to easily see exactly which root was fraudulent inside the fraud prover.